service/main.go

```diff
@@ -43,6 +50,17 @@ func MainContext(ctx context.Context) {
 	params := otelcol.CollectorSettings{
 		Factories: components,
 		BuildInfo: info,
+		ConfigProviderSettings: otelcol.ConfigProviderSettings{
+			ResolverSettings: confmap.ResolverSettings{
+				ProviderFactories: []confmap.ProviderFactory{
+					fileprovider.NewFactory(),
```

Comment:
Is there a reason we're adding all these providers for our distro? Not sure why these are needed now

Reply:
Don't we only use the file provider?

Reply:
There was an upstream change that forces you do explicitly list providers now, I don't have a reference at hand though

Reply:
Ack, it stems from https://github.com/open-telemetry/opentelemetry-collector/pull/9513/files

We used to default to all the default providers so this change makes sense. We should separately reduce this list to just the file provider but that shouldn't block this PR